### PR TITLE
Added some examples

### PR DESCRIPTION
**Pull request by guilhermebodin**

### CODE REVIEW

File: docs/src/examples.md [hunk 1]
```diff
@@ -4,6 +4,136 @@ Here we discuss some useful examples of usage.
...
+    & \min_{x, y, z} & y + z &
+    \\
+    & \;\;\text{s.t.}
+    &x & = 1\\
+    && x & = 1\\
+    &&x & \geq ||(y,z)||_2\\
+\end{align}
+```
+
+The correspondent code in JuMP is
```
Comment (chriscoey) [inline docs/src/examples.md]:
```suggestion
The corresponding code in JuMP is
```

File: docs/src/examples.md [hunk 2]
```diff
@@ -4,6 +4,136 @@ Here we discuss some useful examples of usage.
...
+# Solving a problem the standard way
+model = Model(with_optimizer(ECOS.Optimizer))
+@variable(model, x)
+@variable(model, y)
+@variable(model, z)
+@constraint(model, soccon, [x; y; z] in SecondOrderCone())
+@constraint(model, eqcon, x == 1)
+@objective(model, Min, y + z)
+
+# Solving a problem provinding its dual representation
```
Comment (chriscoey) [inline docs/src/examples.md]:
```suggestion
# Solving a problem by providing its dual representation
```

File: docs/src/examples.md [hunk 3]
```diff
@@ -4,6 +4,136 @@ Here we discuss some useful examples of usage.
...
+    && dualsoccon_2 & = 1\\
+    && dualsoccon_3 & = 1\\
+    &&dualsoccon_1 & \geq ||(dualsoccon_2, dualsoccon_3)||_2\\
+\end{align}
+```
+
+
+## Solving a problem using its dual formulation
+
+It is not clear when solving the dual of a problem is faster than solving its primal but sometimes this feature can be useful.
```
Comment (chriscoey) [inline docs/src/examples.md]: It is solver-dependent

Comment (guilhermebodin) [reply docs/src/examples.md]: Yes, that's worth writing. But even in this case, if you have a solver that only uses the primal to solve the problem, it is not clear when the dual will be faster

File: docs/src/examples.md [hunk 4]
```diff
@@ -4,6 +4,136 @@ Here we discuss some useful examples of usage.
...
+    && dualsoccon_2 & = 1\\
+    && dualsoccon_3 & = 1\\
+    &&dualsoccon_1 & \geq ||(dualsoccon_2, dualsoccon_3)||_2\\
+\end{align}
+```
+
+
+## Solving a problem using its dual formulation
+
+Depending on the solver and on the type of formulation solving the dual of a problem is faster than solving its primal.
```
Comment (chriscoey) [inline docs/src/examples.md]:
```suggestion
Depending on the solver and on the type of formulation, solving the dual problem could be faster than solving the primal.
```

**Pull request merged**